ci-operator/step-registry/openstack/qe/gather/openstack-qe-gather-commands.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Is this file handling any provision? If it is not, maybe the name could be adapted.